### ISSUE TITLE
Map loads off center in 0.11

### ISSUE THREAD

**Issue by DaveSauce** [zcreativelabs/react-simple-maps#69]
When upgrading from `0.10.0` to `0.11.0` or `0.11.1`, my map loads off screen. 

Interestingly, if I update ZoomableGroup center it renders correctly (but uses the same positioning as before - I need to set the value back to get it centered). 

```
<ZoomableGroup
  zoom={mapScale}
  disablePanning={true}
  center={[-97, 38]}
  >
```

On load, I'm seeing this now - (note the first translate value):
`<g class="rsm-zoomable-group" transform: translate(1070.07, 552.33) scale(0.785714, 0.785714) translate(-275, -169)>`

After editing (map appears), or on load in `0.10.0`:
`<g class="rsm-zoomable-group" transform: translate(291.39, 552.33) scale(0.785714, 0.785714) translate(-275, -169)>`

**Comment by zimrick** [comment]
I have a fix for a rotation related bug coming later today (hopfeully). Due to the different way we treat the projection internally as of version 0.11 a small error snuck in there. Not sure whether this will resolve your problem, but it might be related.

**Comment by DaveSauce** [comment]
This is fixed in `0.12.0`. Thanks!
